### PR TITLE
UpdateGuess

### PR DESCRIPTION
**Pull request by Frannyspag24**
update the guess functuon

### CODE REVIEW

**Review by Frannyspag24 — COMMENTED** [review]
guess updated

**Review by JMLovegren — APPROVED** [review]

File: src/SpellingBee/MakePuzzle.py [hunk 1]
```diff
@@ -87,6 +87,8 @@ def checkDataBase(baseWord):
 def choseKeyLetter(uniqueLetters):
     return random.choice(uniqueLetters)
 
+#params: letters is a list of the letters
+#shuffles letters but keeps the center letter at the front of the list
 def shuffle(letters):
```
Comment (JMLovegren) [inline src/SpellingBee/MakePuzzle.py:92]: Need to kill shuffle to resolve merge conflict. Already in Puzzle object

**Review by JMLovegren — CHANGES_REQUESTED** [review]
Remove shuffle from MakePuzzle

**Review by JMLovegren — APPROVED** [review]
That should solve that merge conflict

**Pull request merged**